### DISCUSSION TITLE
How do I backup someone else's organisation/account/repo?

### DISCUSSION THREAD

**Discussion by mariomadproductions** [community/community#54562] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

If I find a project useful, it'd be nice to be able to get a local copy of all the github specific data (issues, discussions, pull requests etc), in addition to getting the actual git data through `git clone --mirror`. How can this be done?

**Comment by GeoffreyFClark** [comment]
You can use this tool to backup a GitHub user or organization: https://github.com/josegonzalez/python-github-backup

It looks like it's using the GitHub REST API and GitHub GraphQL API, which you could also pull data from via your own script as well.

**Reply by mariomadproductions** [reply]
Thanks, I saw that, although looking at the issues it doesn't seem like gets everything. Is there any official github tool or functionality for this?
